testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -25,12 +25,7 @@
 from os import path
 
 import numpy as np
-
-from numpy.testing import (
-    assert_almost_equal,
-    assert_equal,
-    assert_array_almost_equal,
-)
+from numpy.testing import (assert_allclose, assert_equal)
```

Comment:
no longer need the brackets
```suggestion
from numpy.testing import assert_allclose, assert_equal
```